src/main/java/uk/gov/digital/ho/proving/income/audit/ArchiveResource.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.digital.ho.proving.income.audit;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.digital.ho.proving.income.api.RequestData;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static uk.gov.digital.ho.proving.income.api.RequestData.REQUEST_DURATION_MS;
+
+@RestController
+@Slf4j
+public class ArchiveResource {
+
+    private AuditArchiveService auditArchiveService;
+    private RequestData requestData;
+
+    public ArchiveResource(
+        AuditArchiveService auditArchiveService,
+        RequestData requestData
+    ) {
+        this.auditArchiveService = auditArchiveService;
+        this.requestData = requestData;
+    }
+
+    @PostMapping(value = "/archive")
+    public void archive() {
+        log.info("Request received on /archive");
```

Comment:
We should have an Event on all our logs

Reply:
It's on the backlog for this service... but not done yet.